Library/Homebrew/formula.rb

```diff
@@ -1364,6 +1364,18 @@ def self.clear_installed_formulae_cache
     @installed = nil
   end
 
+  def self.comparison
```

Comment:
I think `Comparer` is too generic a class name and it doesn't make sense having a single method there. `tap_and_name_comparison` in `Utils` makes more sense.